job_spec/AUTORIFT.yml

```diff
@@ -32,29 +32,27 @@ AUTORIFT:
               example: LC08_L1GT_118112_20210107_20210107_02_T2
   cost_profiles:
     EDC:
-      cost: 25.0
+      cost: 50.0
     DEFAULT:
       cost: 1.0
   validators: []
   steps:
     - name: ''
       image: ghcr.io/asfhyp3/hyp3-autorift
       command:
-        - ++omp-num-threads
-        - '4'  # 4 vCPUs per 32 GB RAM for the R instance family
+        - Ref::granules
         - --bucket
         - '!Ref Bucket'
         - --bucket-prefix
         - Ref::job_id
         - --parameter-file
-        - '/vsicurl/http://its-live-data.s3.amazonaws.com/autorift_parameters/v001/autorift_landice_0120m.shp'
+        - '/vsicurl/https://its-live-data.s3.amazonaws.com/autorift_parameters/v001/autorift_landice_0120m.shp'
         - --naming-scheme
         - ITS_LIVE_OD
-        - Ref::granules
       timeout: 10800
```

Comment:
Since `timeout` is not changing, I assume runtime hasn't increased significantly?